src/main/kotlin/rhmodding/bread/editor/AnimationsTab.kt

```diff
@@ -321,6 +324,37 @@ open class AnimationsTab<F : IDataModel>(editor: Editor<F>) : EditorSubTab<F>(ed
                             }
                         }
                     }
+                    children += Button("Export as PNG sequence").apply {
+                        disableProperty().bind(disableStepControls)
+                        setOnAction {
+                            val ani = currentAnimation
+                            val directoryChooser = DirectoryChooser()
+                            directoryChooser.title = "Export this animation a sequence of PNG images"
+                            directoryChooser.initialDirectory = editor.dataFile.parentFile
+
+                            val dir = directoryChooser.showDialog(null)
+                            if (dir != null) {
+                                var curStep = 0
+                                var file: File
+                                ani.steps.forEach { step ->
+                                    curStep++
+                                    file = File("${dir.toString()}${File.separator}$curStep.png")
```

Comment:
Should we consider padding the `curStep` with zeroes in the filename? It might make it easier for File Explorer to sort.

Reply:
Sounds good. A length of 3 would be enough, I assume? I honestly can't imagine an animation going over 999 steps lol

Reply:
(by "length of 3" I mean "001", "002", "050", "100"...)

Reply:
IMO I think it should be dynamic and 3 digits minimum.

Quick hacky impl:
```kotlin
val padCount: Int = ani.steps.size.toString().size.coerceAtLeast(3)
ani.steps.forEachIndexed { curStep, step ->
  val stepNum: String = curStep.toString().padStart(padCount, padChar = '0')
  // ... use stepNum in the filename
}

Reply:
Something I missed was the `file` instantiation: It may be better to use the Kotlin standard library function `File.resolve(subpath)` since it handles the system file separator and other edge cases more robustly.

```kotlin
file = dir.resolve(stepFilename)
```